src/xo-app/new/import/index.css

```diff
@@ -0,0 +1,32 @@
+.dropzone {
+    border-color: #8f8686;
+    border-radius: 4px;
+    border-style: dashed;
+    border-width: 2px;
```

Comment:
You can use the border property as a shortcut for `<border-width> <border-style> <border-color>`.